libs/input-mapping/src/Reader.php

```diff
@@ -86,7 +86,9 @@ public function downloadTables(
         );
         $tablesDefinition = $tableResolver->resolve($tablesDefinition);
         $strategy = $this->strategyFactory->getTableInputStrategy($stagingType, $destination, $tablesState);
-        if ($readerOptions->devInputsDisabled()) {
+        if ($readerOptions->devInputsDisabled()
```

Comment:
Ješte fix toho, že zakazují devInputs - nejde použít, protože devInputs jsou stejný jako prodInputs, ergo začne to řvát vždycky, když začne existovat tabulka v branchi.

Problém, kterej tohle řešilo byl, aby si někdo do inputu nedal `in.c-444444-bucket`, kterej by po mergi do produkce nefungoval. Vzhledem k tomu, že při useBranchStorage jsou názvy stejný, tak tohle nehrozí a tímpádem je devInputDisabled feature irrelevantní.